config/Betterloadingscreen/tips/en_US.txt

```diff
@@ -12,7 +12,7 @@ Material tiers play a role when breaking pipes
 If a machine catches fire, it can explode
 Adding water to an empty but hot Boiler will make a loud bang!
 Avoid Eldritch Obelisks!
-You can bind the questbook to a key to save you from using the item
+You can bind the questbook to a key to save you from using a slot
```

Comment:
'the item' was very much correct.

Reply:
maybe 'to free up a slot' then?

Reply:
you're either having a book in your hotbar or not, so for me it's a way to free up a precious (hotbar) slot

Reply:
you can put items in storage. anyway, whatever